weave/trace_server/trace_server_interface.py

```diff
@@ -423,6 +423,13 @@ class CallsQueryReq(BaseModel):
         examples=[["inputs.self.message", "inputs.model.prompt"]],
         description="Columns to expand, i.e. refs to other objects",
     )
+    # Internal flag used for playing nice with the recursive object ref handling
```

Comment:
this isn't really internal-only as it is used by the UI client and potentially python clients in the future. You basically are saying we have two modes:
1. expand_columns is only used to specify filter/sort expansion
2. expand_columns does 1 AND returns these expanded values.

where today we have: 3: expand the values.

I think you might want to call this: `return_expanded_column_values` or something that more clearly describes the purpose of the field